javacutil/src/org/checkerframework/javacutil/Resolver.java

```diff
@@ -81,6 +85,22 @@ public Resolver(ProcessingEnvironment env) {
             err.initCause(e);
             throw err;
         }
+
+        try {
+            ACCESSERROR = Class.forName("com.sun.tools.javac.comp.Resolve$AccessError");
+            ACCESSERROR_ACCESS = ACCESSERROR.getMethod("access", Name.class, TypeSymbol.class);
+            ACCESSERROR_ACCESS.setAccessible(true);
+        } catch (ClassNotFoundException e) {
+            Error err = new AssertionError(
```

Comment:
Instead of re-throwing the error, the error message would be cleaner if you used `ErrorReporter.errorAbort`. This is an expected error, so you shouldn't use exceptions more than necessary.